app/__init__.py

```diff
@@ -1,5 +1,11 @@
 from flask import Flask
+from app.config import Config
```

Comment:
similar changes were not done by me. These changes are from bloggy dir. Now we have app instead of bloggy so these changes were automatically detected by git